src/main/java/uk/gov/companieshouse/alphabeticalcompanysearchconsumer/util/ApiClientUtils.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.companieshouse.alphabeticalcompanysearchconsumer.util;
+
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+import uk.gov.companieshouse.stream.ResourceChangedData;
+import java.io.IOException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+
+public class ApiClientUtils {
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    public static CompanyProfileApi mapMessageToRequest(ServiceParameters parameters) {
+        ResourceChangedData data = parameters.getData();
+        String jsonData = data.getData();
+        
+        try {
+            return objectMapper.readValue(jsonData, CompanyProfileApi.class);
+        } catch (IOException e) {
+            // Handle the exception appropriately
+            e.printStackTrace();
```

Comment:
Ideally we'd want to stick to using a logger if possible when printing things, and the whole stacktrace could include a lot unnecessary information.

Reply:
could you instead use the getRootCause in the LoggingUtils?